### PR TITLE
Enable prx obs parser by default

### PR DESCRIPTION
**Pull request by jtec**
This PR enables the faster prx obs parser.

### CODE REVIEW

**Comment by jtec** [comment]
@plutonheaven Ideas for unit tests we should add for the parser?

**Comment by plutonheaven** [comment]
> @plutonheaven Ideas for unit tests we should add for the parser?

Not really. Just draw random observables and enter them manually in a test...?

**Comment by plutonheaven** [comment]
@jtec I wrote a dumb test...

File: src/prx/rinex_obs/test/test_parser.py [hunk 1]
```diff
@@ -22,3 +45,75 @@ def test_compare_to_georinex():
         .reset_index(drop=True)
     )
     assert georinex_output.equals(prx_output)
+
+
+def test_basic_check_on_rinex(input_for_test_tlse):
```
[the remark this answers is not in the document]
Comment (plutonheaven) [reply src/prx/rinex_obs/test/test_parser.py:50]: Let's go

**Review by plutonheaven — APPROVED** [review]

**Pull request merged**